src/SampleLibraryTests/TestServices.dll.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-  <appSettings>
-    <add key="DynamoBasePath" value=""/>
-    <add key="RequestedLibraryVersion" value="Version223"/>
-  </appSettings>
+    <appSettings>
+        <add key="DynamoBasePath" value="D:\dev\dynamo\DynamoDS\bin\AnyCPU\debug"/>
```

Comment:
Obviously, this is not meant for CICD, is there CICD setup for this project?

Reply:
removed